<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMDProjectParaForm</class>
 <widget class="QWidget" name="CMDProjectParaForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="enableLoginBeforeRunCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>登录后才能运行项目</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableSPCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>启用安全保护</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>430</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="spTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>431</width>
     <height>190</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <column>
    <property name="text">
     <string>动作</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>安全策略</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
